kong/plugins/opentelemetry/otlp.lua

```diff
@@ -72,9 +77,39 @@ local function transform_events(events)
   return pb_events
 end
 
+-- this function is to translate the span to otlp span, but of universal span format
+local function translate_span(span)
```

Comment:
This comment + function name don't clarify what the function is doing. Since it only truncates/elongates the trace id: